tests/output_filters/related_files_test.py

```diff
@@ -1,59 +1,89 @@
+
+        Args:
+            input_blob: An enumerable of dicts
+            when: A callable when to init the RelatedFilesFilter with
+            initial_terms: An enumerable of strings to init the RelatedFilesFilter with
+            expected_terms: The expected final value of RelatedFilesFilter.terms
+            expected_usernames: The expected final value of RelatedFilesFilter.usernames
+            expected_is_related: An enumerable of the expected value of 'osxcollector_related' for each output_blob
+        """
+
+        def create_related_files_filter():
+            self._output_filter = RelatedFilesFilter(when=when, initial_terms=initial_terms)
+            return self._output_filter
+
+        output_blobs = self.run_test(create_related_files_filter, input_blobs=input_blobs)
+        if expected_terms:
+            assert_equal_sorted(expected_terms, self._output_filter.terms)
+        if expected_usernames:
+            assert_equal_sorted(expected_usernames, self._output_filter.usernames)
+        if expected_is_related:
+            self.assert_key_added_to_blob('osxcollector_related', expected_is_related, input_blobs, output_blobs)
+        return output_blobs
+
+
+class CreateTermsTest(RelatedFilesFilterTest):
+
+    """Focuses on testing that terms are properly created."""
 
     def test_single_term(self):
         initial_terms = ['one_word']
         expected = ['one_word']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_multi_terms(self):
         initial_terms = ['one_word', 'pants', 'face']
         expected = ['one_word', 'pants', 'face']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_split_terms(self):
         initial_terms = ['/ivanlei/source/osxcollector']
         expected = ['ivanlei', 'source', 'osxcollector']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_whitelist_terms(self):
         initial_terms = ['/Users/ivanlei/source/osxcollector', '/Users/ivanlei/virtual_envs/osxcollector/bin/python']
         expected = ['ivanlei', 'source', 'osxcollector', 'virtual_envs']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_whitelist_username_terms(self):
         initial_terms = ['/Users/ivanlei/source/osxcollector', '/Users/ivanlei/virtual_envs/osxcollector/bin/python']
         expected = ['source', 'osxcollector', 'virtual_envs']
         blob = {'osxcollector_username': 'ivanlei'}
 
-        self._test_create_terms(initial_terms, expected, blob)
+        self._run_test(input_blobs=[blob], initial_terms=initial_terms, expected_terms=expected, expected_usernames=['ivanlei'])
```

Comment:
Super minor, but you might want to move "['ivanlei']" under "blob = ..." for consistency.